Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,247 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
 | Heerlen           |
 | Alkmaar           |
 +-------------------+
-28 rows in set (0.00 sec)
\ No newline at end of file
+28 rows in set (0.00 sec)
+2.What is the population of Rotterdam ?
+
+ SELECT name, population FROM city WHERE Name='Rotterdam';
++-----------+------------+
+| name      | population |
++-----------+------------+
+| Rotterdam |     593321 |
++-----------+------------+
+1 row in set (0.00 sec)
+
+3.What's the name of all the countries on the continent ‘North America’ ?
+
+SELECT Name FROM country WHERE country.continent='North America';
```

Comment:
:+1:

Reply:
Careful - `country.continent` may not be the same as `country.Continent`. Uppercase/lowercase matters.